yapapi/payload/vm.py

```diff
@@ -24,8 +24,9 @@
 from yapapi.props import inf
 from yapapi.props.inf import InfBase, INF_CORES, RUNTIME_VM, ExeUnitRequest, ExeUnitManifestRequest
 
-_DEFAULT_REPO_SRV: Final = "_girepo._tcp.dev.golem.network"
-_FALLBACK_REPO_URL: Final = "http://girepo.dev.golem.network:8000"
+_DEFAULT_REPO_SRV: Final[str] = "_girepo._tcp.dev.golem.network"
+_FALLBACK_REPO_URL: Final[str] = "http://girepo.dev.golem.network:8000"
+_DEFAUTL_TIMEOUT_SECONDS: Final[int] = 10
```

Comment:
```suggestion
_DEFAULT_TIMEOUT_SECONDS: Final[int] = 10
```

Reply:
and, actually, I'd remove "SECONDS" since timeouts _usually_ are expressed in seconds

Reply:
Some libs are using milliseconds and `srvresolver` doesn't specified unit. I needed to go down to `socket` docs to find out. Including units just make sense for me, I would consider one of those 3 options:
1. In variable name
```_DEFAULT_TIMEOUT_SECONDS: Final[int] = 10```

2. Type alias
```Second = int
_DEFAULT_TIMEOUT: Final[Second] = 10```

3. In comment
_DEFAULT_TIMEOUT: Final[int] = 10  # secound```

Let me know what do you think

Reply:
Thinks I am not sure about are:
1. is `DEFAULT_TIMEOUT` a good name
2. is this the right place to use it
3. do we want have this in a configuration so user can change default to its own preferred value via env variable or via argument

Reply:
okay, let's leave `DEFAULT_TIMEOUT_SECONDS`